src/main/java/hybrid/it/internship/library/service/impl/BookService.java

```diff
@@ -0,0 +1,41 @@
+package hybrid.it.internship.library.service.impl;
+
+import hybrid.it.internship.library.entity.Book;
+import hybrid.it.internship.library.repository.BookRepository;
+import hybrid.it.internship.library.service.IBookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class BookService implements IBookService {
+
+    @Autowired
```

Comment:
Don't use @Autowired, use constructor injection instead(@AllArgsConstructor). Also fields should be final.